cameraserver/build.gradle

```diff
@@ -9,6 +9,14 @@ evaluationDependsOn(':hal')
 
 apply from: "${rootDir}/shared/javacpp/setupBuild.gradle"
 
+checkstyle {
+    checkstyleMain {
+        // This is required because of a too long line in vision/package-info.java.
+        // Changing the length of this line changes the semantics of the doc.
+        exclude '**/package-info.java'
+    }
+}
+
```

Comment:
Instead of excluding the file entirely, can you just disable line length checks?

Add this to `styleguide/suppressions.xml`:

```
<suppress files="package-info.java"
            checks="LineLength" />
```

Reply:
But that would disable the checks on all package-info.java files. Although maybe that is better.

Reply:
Ok fixed